llvm/docs/ReleaseNotes.md

```diff
@@ -304,6 +304,10 @@ Changes to LLDB
     stop reason = SIGSEGV: sent by tkill system call (sender pid=649752, uid=2667987)
   ```
 * ELF Cores can now have their siginfo structures inspected using `thread siginfo`.
+* Disassembly of unknown instructions now produces "<unknown>" instead of
+  nothing at all
+* Changed the format of opcode bytes to match llvm-objdump when disassembling
+  RISC-V code with disassemble's --byte option.
```

Comment:
Put the command name and option in Markdown's plain text format with backticks. So it looks like `disassemble`'s `--byte` option.